src/main/java/mate/academy/springboot/datajpa/controller/CategoryController.java

```diff
@@ -0,0 +1,54 @@
+package mate.academy.springboot.datajpa.controller;
+
+import lombok.RequiredArgsConstructor;
+import mate.academy.springboot.datajpa.dto.request.CategoryRequestDto;
+import mate.academy.springboot.datajpa.dto.response.CategoryResponseDto;
+import mate.academy.springboot.datajpa.model.Category;
+import mate.academy.springboot.datajpa.services.CategoryService;
+import mate.academy.springboot.datajpa.services.mapper.CategoryMapper;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/category")
+@RequiredArgsConstructor
+public class CategoryController {
+    private final CategoryService categoryService;
+    private final CategoryMapper categoryMapper;
+
+    @PostMapping
+    public CategoryResponseDto create(@RequestBody
+                                      CategoryRequestDto categoryRequestDto) {
+        return categoryMapper.toDto(
+                categoryService.create(
+                        categoryMapper.toModel(categoryRequestDto)));
+    }
+
+    @GetMapping("/{id}")
+    public CategoryResponseDto getById(@PathVariable Long id) {
+        return categoryMapper.toDto(categoryService.getById(id));
+    }
+
+    @DeleteMapping("/{id}")
+    @ResponseStatus(HttpStatus.OK)
```

Comment:
do you need this annotation?

Reply:
used for information that everything worked ok

Reply:
you will have such a status in a response by default